### PR TITLE
Create virtualenvwrapper-4.8.2-r1.ebuild

### PR DESCRIPTION
**Pull request by rain0r**
Added python target 3.7

### CODE REVIEW

**Comment by prometheanfire** [comment]
This would probably be a good time to update to 4.8.4 and EAPI 7 as well.  It doesn't look like much changed as far as deps go, so should be easy (no docs build for us).

https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/branches/compare/4.8.4%0D4.8.2#diff

**Comment by rain0r** [comment]
> 
> 
> This would probably be a good time to update to 4.8.4 and EAPI 7 as well. It doesn't look like much changed as far as deps go, so should be easy (no docs build for us).
> 
> https://bitbucket.org/virtualenvwrapper/virtualenvwrapper/branches/compare/4.8.4%0D4.8.2#diff

Added `virtualenvwrapper-4.8.4.ebuild` with `EAPI=7`.

Should I remove `virtualenvwrapper-4.8.2-r1.ebuild`?

**Comment by prometheanfire** [comment]
ya, and squash the commits, thanks

**Comment by rain0r** [comment]
> 
> 
> ya, and squash the commits, thanks

Done; hope I didn't mess anything up.

**Comment by prometheanfire** [comment]
https://github.com/gentoo/gentoo/pull/12617/commits shows that it's still two commits, those will need to be squashed.  You'll also need to read https://github.com/gentoo/gentoo/pull/12617/commits#issuecomment-518000533 (and it's links) and sign the commit if you wish to continue.  Thanks

File: dev-python/virtualenvwrapper/virtualenvwrapper-4.8.2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline dev-python/virtualenvwrapper/virtualenvwrapper-4.8.2-r1.ebuild]: please bump to 7

File: dev-python/virtualenvwrapper/virtualenvwrapper-4.8.2-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+inherit distutils-r1
+
+DESCRIPTION="Set of extensions to Ian Bicking's virtualenv tool"
+HOMEPAGE="https://bitbucket.org/dhellmann/virtualenvwrapper
+	https://pypi.org/project/virtualenvwrapper/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (Zlogene) [inline dev-python/virtualenvwrapper/virtualenvwrapper-4.8.2-r1.ebuild]: do not commit straightto stable

File: dev-python/virtualenvwrapper/virtualenvwrapper-4.8.2-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+IUSE="test"
```
Comment (Zlogene) [inline dev-python/virtualenvwrapper/virtualenvwrapper-4.8.2-r1.ebuild]: you do not use this flag at all.

**Comment by prometheanfire** [comment]
looks like your SOB line needs to be fixed, but that's it.

`Signed-off-by: Rainer Hihn hihn.rainer@gmail.com`
should be
`Signed-off-by: Rainer Hihn <hihn.rainer@gmail.com>`

**Comment by rain0r** [comment]
> 
> 
> looks like your SOB line needs to be fixed, but that's it.
> 
> `Signed-off-by: Rainer Hihn hihn.rainer@gmail.com`
> should be
> `Signed-off-by: Rainer Hihn <hihn.rainer@gmail.com>`

Any action required by me?

**Comment by prometheanfire** [comment]
the commit needs to be amended to change the signed-off-by line in the commit message

**Comment by rain0r** [comment]
> 
> 
> the commit needs to be amended to change the signed-off-by line in the commit message

Sorry, but isn't it already changed?

https://github.com/gentoo/gentoo/pull/12617/commits/3596fad5b7973790465e9597fe159ae744ba313f

grafik

**Comment by prometheanfire** [comment]
ok, merged now, thanks

**Pull request closed without merging**